static-build/cmake/Package.cmake

```diff
@@ -0,0 +1,120 @@
+# CPack helps to create packages for deb and rpm based distros. 
+# Creating rpm packages depends on package rpm-build. Deb package doesn't
+# need any additional packages. CPack should be version 3.1 or greater.
+# For building packages, you should use centos 7 for save compatibility.
+# cmake3 and cpack3 have bug in centos 7 which add build artifacts to package.
+# To avoid this bug we create 3 packages: tarantool, tarantool-devel and
+# tarantool-slim. The first package contains only binary files and building artifacts. 
+# Second package contains only header files. Third package contains only binary
+# file tarantool.
+
+message(VERSION " $ENV{VERSION}")
+
+message(ARCHITECTURE " ${CMAKE_SYSTEM_PROCESSOR}")
+
+# Set architecture for packages (x86_64 or aarch64)
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
+        set(CPACK_RPM_PACKAGE_ARCHITECTURE "x86_64")
+elseif(ARCHITECTURE STREQUAL "aarch64\n")
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "arm64")
+        set(CPACK_RPM_PACKAGE_ARCHITECTURE "aarch64")
+else()
+        message(FATAL_ERROR "Unsupported architecture ${ARCHITECTURE}")
+endif()
+
+message(CPACK_DEBIAN_PACKAGE_ARCHITECTURE " ${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
+message(CPACK_RPM_PACKAGE_ARCHITECTURE " ${CPACK_RPM_PACKAGE_ARCHITECTURE}")
+
+set(CPACK_COMPONENTS_GROUPING ONE_PER_GROUP)
+set(CPACK_RPM_COMPONENT_INSTALL ON)
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+
+install(DIRECTORY tarantool-prefix/bin/ DESTINATION bin
+        USE_SOURCE_PERMISSIONS
+        COMPONENT tarantool
+        EXCLUDE_FROM_ALL)
+
+install(DIRECTORY tarantool-prefix/include/ DESTINATION include
+        USE_SOURCE_PERMISSIONS
+        COMPONENT devel
+        EXCLUDE_FROM_ALL)
+
+install(DIRECTORY tarantool-prefix/bin/ DESTINATION bin
+        USE_SOURCE_PERMISSIONS
+        COMPONENT slim
+        EXCLUDE_FROM_ALL
+        FILES_MATCHING PATTERN "tarantool")
+
+set(CPACK_GENERATOR "DEB;RPM")
+set(CPACK_PACKAGE_NAME "tarantool")
+set(CPACK_PACKAGE_VERSION  $ENV{VERSION})
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_CONTACT "tarantool@tarantool.io")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.tarantool.io")
```

Comment:
IMHO, it would be better to use `https://www.tarantool.io/en/`.

Reply:
Actually not.

Reply:
BTW, shouldn't it be `https://tarantool.org/`? I can see this address in debian/control and rpm/tarantool.spec.

Reply:
fixed